pytplot/__init__.py

```diff
@@ -220,6 +222,13 @@ def link_to_tvar(self, name, link, method='linear'):
 #from .clip import clip
 #from .deflag import deflag
 
+# Add in default values for crosshair names, y/z axis_opt, etc.
+default_values = {'x_crosshair_name': 'X',
+                  'y_crosshair_name': 'Y',
+                  'z_crosshair_name': 'Z',
+                  'y_axis_type': 'linear',
+                  'z_axis_type': 'linear'}
+
```

Comment:
What I was kinda thinking was the TVar class would just automatically set things like these, rather than have them outside of the class.  That way, a statement like:

pytplot.data_quants[self.tvar_name].xaxis_opt['crosshair']

Would never throw an error, since xaxis_opt would always have the key "crosshair" inside of it.